src/layouts/Sidebar.tsx

```diff
@@ -675,7 +678,7 @@ const Sidebar = () => {
               onClickAnnouncementsButton={toggleIsAnnouncementsModalVisible}
               onClickJobLoggerButton={toggleIsJobLoggerVisible}
             />
-            <ProfileMenu />
+            {isClient && <ProfileMenu />}
```

Comment:
We should check we the `ProfileMenu` is only available on the client side, ideally it should render both on client and server. 
This condition is a bit of a hack, and I want to avoid things like on uiv.